hexrd/ui/resources/calibration/default_instrument_config.yml

```diff
@@ -1,6 +1,7 @@
 beam:
   energy: 71.6764
   vector: {azimuth: 90.0, polar_angle: 90.0}
+  source_distance: .inf
```

Comment:
Strictly shouldn't need this, since the instrument class has the default.  But it doesn't hurt.  You verified that `.inf` is the yaml spec for IEEE infinity?

Reply:
Yes I did. It is working.